README.md

```diff
@@ -75,9 +75,11 @@ docker run -it \
 > Choose the tag that best suits your needs based on stability requirements and desired features.
 
 You'll find OpenDevin running at [http://localhost:3000](http://localhost:3000) with access to `./workspace`. To have OpenDevin operate on your code, place it in `./workspace`.
-
 OpenDevin will only have access to this workspace folder. The rest of your system will not be affected as it runs in a secured docker sandbox.
 
+Upon opening OpenDevin, you must select the appropriate `Model` and enter the `API Key` within the settings that should pop up automatically. These can be set at any time by selecting
+the `Settings` button (gear icon) in the UI.
+
```

Comment:
We can add that: if the model you wish is not in the list, you can enter it in the text box manually, and just save it.

Reply:
Updated. Many thanks for the suggestion.